### PR TITLE
[14.0][ADD] sale_stock_on_hand_popup

### PR DESCRIPTION
**Pull request by renda-dev**
Add a "View Stocks" button in the "Availability" widget of the sale order line.

### CODE REVIEW

File: sale_stock_on_hand_popup/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This module extends the functionality of 'Sale Stock' module to add the 'View Stock'
+button that redirects the user to the 'Update Quantity' page of the selected product.
```
Comment (francesco-ooops) [inline sale_stock_on_hand_popup/readme/DESCRIPTION.rst]: This module allows user to check from sale order line all inventory lines related to that product.

File: sale_stock_on_hand_popup/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
+To use this module, you need to:
+
+#. Go to *Sales > Orders > quotations* and create a new one.
+#. Add a sale order line with a storable product
+#. Click on the icon in the line and you will
+   see in the popover the '-> View Stock' button that will redirect
+   the user to the 'Update Quantity' page of that certain product
```
Comment (francesco-ooops) [inline sale_stock_on_hand_popup/readme/USAGE.rst]: the user to a 'Stock Lines' page for the specific product.

**Review by francesco-ooops — APPROVED** [review]
Functional ok!

**Review by HekkiMelody — APPROVED** [review]

**Comment by francesco-ooops** [comment]
@rousseldenis would you mind merging? :)

**Comment by rousseldenis** [comment]
@renda-dev @francesco-ooops I'm just wondering if this should go or not in sale repo as the entry point is sale order

**Comment by francesco-ooops** [comment]
Hi @rousseldenis the rationale was that this module is quite a clone of https://github.com/OCA/stock-logistics-availability/tree/16.0/sale_stock_available_info_popup, and this is the "parent" repo for v14 of stock-logistics-availability

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**